### ISSUE TITLE
Translation parsing problems

### ISSUE THREAD

**Issue by fenume** [marius00/iagd#70]
IA fail to get names for some items when i use latest russian translation(r96), they appear as Unknown, but right name shown below with different translation tags [ns],[ms], etc
Screenshot: https://i.imgur.com/fomXJtD.png

I cleared database parsed data again and update items few times, with GD running and not - didn't help.

**Comment by marius00** [comment]
Interesting.. so the name blow is the correct one?

(Also those are recipes, items you can craft, not ones you own)

**Comment by fenume** [comment]
Yes, the names below is correct, but prepended with [gender codes](https://www.grimdawn.com/forums/showthread.php?t=72223) without brackets.
I know this is recipes, but i don't have enough items to check this bug on them. 
Affected recipes count:
[ms] 60
[fs] 20
[ns] 13
[mp] 5
[fp] 0
[np] 7
